src/result_object/index.js

```diff
@@ -1,9 +1,8 @@
 // @flow
-import fail, { FAIL } from './helpers/fail';
 
-declare type ErrorAndWarningObject = {
-    [name: string]: Array<string>
-};
+export const WARN: Severity = 'warn';
+export const FAIL: Severity = 'fail';
+const severities: Array<Severity> = [WARN, FAIL];
 
 /** Class representing validation state. */
 class ResultObject {
```

Comment:
don't need to return this in constructor

Reply:
very true